src/main/kotlin/io/emeraldpay/dshackle/quorum/MaximumValueQuorum.kt

```diff
@@ -0,0 +1,57 @@
+package io.emeraldpay.dshackle.quorum
+
+import io.emeraldpay.dshackle.upstream.Upstream
+import io.emeraldpay.dshackle.upstream.signature.ResponseSigner
+import io.emeraldpay.etherjar.hex.HexQuantity
+import java.util.concurrent.atomic.AtomicReference
+
+class MaximumValueQuorum : CallQuorum, ValueAwareQuorum<String>(String::class.java) {
+    private val max: AtomicReference<Long?> = AtomicReference()
```

Comment:
I think we can use just `Long`, there is no any parallel work

Reply:
in broadcast reader its parallel